corgie/cli/compute_stats.py

```diff
@@ -21,6 +21,8 @@
 
 
 class ComputeStatsJob(scheduling.Job):
+    LAYERS = {}
```

Comment:
What made you do this? Was one worker doing most of these jobs?

Reply:
Talking with @nkemnitz, this was because every new job would ping for a layer's info file in case it needed to be created. That was a substantial overhead.